ian-page.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Ian's Dumpster Fire of a Page</title>
+  <link rel="stylesheet" href="main.css" type="text/css" />
+</head>
+
+<body>
+  <div class="centered-content">
+    <h1>Ian's Page</h1>
+    <h3 style="margin-bottom: 0;">Things I won't do</h3>
+    <div>
+      <ol>
+        <li>Give you up</li>
+        <li>Let you down</li>
+        <li>Run around and desert you</li>
+        <li>Make you cry</li>
+        <li>Say goodbye</li>
+        <li>Tell a lie and hurt you</li>
+      </ol>
+    </div>
```

Comment:
Now if only the YouTube video was embedded here...

Reply:
something about youtube doesn't allow embedded videos... something with youtube itself unless the video permits. unless there is some code that proves me wrong

Reply:
Yeah, it was the specific video you were trying to embed. Must have been disallowed by the uploader. This one embeds just fine though:
```
<iframe width="560" height="315" src="https://www.youtube.com/embed/dQw4w9WgXcQ?autoplay=1" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```